### PR TITLE
Ada1 8

### PR DESCRIPTION
**Pull request by GabrielParle**
search Improved.

### CODE REVIEW

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx [hunk 1]
```diff
@@ -113,4 +133,39 @@ export default class LocationReportsSearch extends React.Component {
...
+                console.log(resultarr[0]);
+                this.setState({ locationsArr : resultarr});
+                console.log(this.setState.locationsArr[0]);
+            });
+
+        } catch (error) {
+            return this.setState({ message: { message: error.message, type: 'danger' } });
+        }
+    }
+
```
Comment (ch55secake) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx:168]: trailing blank lines

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx [hunk 2]
```diff
@@ -27,36 +30,53 @@ export default class LocationReportsSearch extends React.Component {
...
-                            value={this.state.callSign}
-                            onChange={this.onCallSignChange}/>
+                        <FormControl  onLoad={this.loadCallSigns} type="text" list="data" onChange={this._onChange} />
+                        
+                        <datalist id="data">
+                            {this.state.callSigns.map((item,index) =>
+                                <option key={index} value={item.callSign} />
+                            )}
+                        </datalist>
+
```
Comment (ch55secake) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx:64]: blank lines

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx [hunk 3]
```diff
@@ -27,36 +30,53 @@ export default class LocationReportsSearch extends React.Component {
         this.onFromChange = this.onFromChange.bind(this);
         this.onToChange = this.onToChange.bind(this);
         this.onSubmit = this.onSubmit.bind(this);
+        this.loadCallSigns = this.loadCallSigns(this);
+        this.loadLocations = this.loadLocations(this);
     }
 
+
```
Comment (ch55secake) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx:37]: blank lines

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx [hunk 4]
```diff
@@ -27,36 +30,53 @@ export default class LocationReportsSearch extends React.Component {
...
         this.onSubmit = this.onSubmit.bind(this);
+        this.loadCallSigns = this.loadCallSigns(this);
+        this.loadLocations = this.loadLocations(this);
     }
 
+    
+    
+
     render() {
+
```
Comment (ch55secake) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-location-reports.jsx:41]: loads here and below

**Review by ch55secake — COMMENTED** [review]
looks good just delete blank lines

**Comment by GabrielParle** [comment]
okay done

**Pull request merged**